packages/codegen/package.json

```diff
@@ -20,7 +20,7 @@
     "@dprint/json": "0.15.6",
```

Comment:
I'm not sure why this was reverted a while back: https://github.com/joist-orm/joist-orm/commit/546f346f2a511e8381545805bc09452888ce50ec

It would be nice to also loosen this requirement if possible.

Reply:
Er, yeah really should have put a "why" into that revert--not remembering the issue